models/compat/wakatime/v1/leaders.go

```diff
@@ -0,0 +1,97 @@
+package v1
+
+import (
+	"github.com/duke-git/lancet/v2/slice"
+	"github.com/muety/wakapi/models"
+	"time"
+)
+
+// partially compatible with https://wakatime.com/developers#leaders
+
+type LeaderboardViewModel struct {
+	CurrentUser CurrentUser        `json:"current_user"`
+	Data        []LeaderboardEntry `json:"data"`
+}
+
+type CurrentUser struct {
+	Rank int   `json:"rank"`
+	Page int   `json:"page"`
+	User *User `json:"user"`
+}
+
+type LeaderboardEntry struct {
+	Rank         int          `json:"rank"`
+	RunningTotal RunningTotal `json:"running_total"`
+	User         *User        `json:"user"`
+}
+
+type RunningTotal struct {
+	TotalSeconds              float64    `json:"total_seconds"`
+	HumanReadableTotal        string     `json:"human_readable_total"`
+	DailyAverage              float64    `json:"daily_average"`
+	HumanReadableDailyAverage string     `json:"human_readable_daily_average"`
+	Languages                 []Language `json:"languages"`
+}
+
+type Language struct {
+	Name         string  `json:"name"`
+	TotalSeconds float64 `json:"total_seconds"`
+}
+
+func NewRunningTotal(entry *models.LeaderboardItemRanked) RunningTotal {
+	return RunningTotal{
+		TotalSeconds:              entry.Total.Seconds(),
+		HumanReadableTotal:        entry.Total.String(),
```

Comment:
Minor: please use `helpers.FmtWakatimeDuration` for compatibility with the official API and for consistency with the other endpoints. Also, please always round to integer precision (`daily_average: 4664.142857142857` looks a bit odd).